apps/design-land/src/app/card/examples/card-with-color/card-with-color.component.html

```diff
@@ -0,0 +1,12 @@
+<daff-card [color]="colorControl.value">
+  <img daffCardImage src="/assets/card/uber-logo-dark.svg" />
+  <h4 daffCardTitle>Title</h4>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+  Some other content
+</daff-card>
+
+<select [formControl]="colorControl">
```

Comment:
Where is `black`, `white`, `theme`, and `theme-contrast`?

Reply:
I added those examples back in e719c729775289516cdff0a749bdff3bdff92295

Reply:
Why do they need to be in separate components? Can't they rendered using the `card-with-color` component?

Reply:
I addressed this.